README.md

```diff
@@ -1,61 +1,80 @@
-# sed
-[![Documentation Status](https://github.com/OpenCOMPES/sed/actions/workflows/build_deploy_docs.yml/badge.svg)](https://opencompes.github.io/sed/)
+
+[![Documentation Status](https://github.com/OpenCOMPES/sed/actions/workflows/documentation.yml/badge.svg)](https://opencompes.github.io/sed/)
 ![](https://github.com/OpenCOMPES/sed/actions/workflows/linting.yml/badge.svg?branch=main)
 ![](https://github.com/OpenCOMPES/sed/actions/workflows/testing_multiversion.yml/badge.svg?branch=main)
-![](https://img.shields.io/pypi/pyversions/sedprocessor)
-![](https://img.shields.io/pypi/l/sedprocessor)
-![](https://img.shields.io/pypi/v/sedprocessor)
+![](https://img.shields.io/pypi/pyversions/sed-processor)
+![](https://img.shields.io/pypi/l/sed-processor)
+![](https://img.shields.io/pypi/v/sed-processor)
 [![Coverage Status](https://coveralls.io/repos/github/OpenCOMPES/sed/badge.svg?branch=main&kill_cache=1)](https://coveralls.io/github/OpenCOMPES/sed?branch=main)
 
 
 Single Event Data Frame Processor: Backend to handle photoelectron resolved datastreams
 
 # Installation
 
-## Conda approach
+## Pip (for users)
+
+- Create a new virtual environment using either venv, pyenv, conda etc. See below for example.
+
+- Install sed, distributed as sed-processor on PyPI
 
-Clone this repository and cd to its root folder.
-Create a new environment by typing:
 ```
-conda env create -f env.yml
+pip install sed-processor
 ```
-This should install all the requirements to run `sed` in your environment.
-To activate your environment:
+- This should install all the requirements to run `sed` in your environment.
+
+- If you intend to work with jupyter notebooks, it is helpfull to install a jupyter kernel of your environment. This can be done, once activating your environment, by typing:
+
 ```
-conda activate sed_conda
+python -m ipykernel install --user --name=sed_kernel
 ```
-If you intend to work with jupyter notebooks, it is helpfull to install a jupyter kernel of your environment. This can be done, once activating your environment, by typing:
+
+- To create an environment using venv, use
+
+```
+python -m venv .sed-venv
+```
+
+- To activate your environment:
+
 ```
-python -m ipykernel install --user --name=sed_conda
+source .sed-venv/bin/activate
 ```
 
```

Comment:
I would add a "bleeding edge" installation, where we propose to run `run pip install -e .` in the main repo folder to install it interactively. This would be the preferred way when working in beamtimes with developers, as certainly a new branch with quick implementation of feature requests will spawn

Reply:
I think then we can come up with three categories of people using sed:

- Users
- Contributors
- Maintainers (This doesn't have to be in README but only in documentation)

And contributors can have the bleeding edge installation. However, if they want to create docs or run tests locally, they can't without using poetry (but that's not an issue I guess due to all the github checks we have)

I am not sure how contributors work so could you write a small section and I can just push that?

Reply:
I have pushed some changes to README. You can edit if you wish

Reply:
This is good for now. I will see what the feedback is next week and modify accordingly.